flocker/provision/_aws.py

```diff
@@ -51,6 +53,32 @@
 INSTANCE_TIMEOUT = 300
 
 
+class EliotLogHandler(logging.Handler):
+    # Whitelist ``"msg": "Params:`` field for logging.
+    _to_log = {"Params"}
+
+    def emit(self, record):
+        fields = vars(record)
```

Comment:
I am not sure (because someone else wrote the code). Maybe we still want `fields`, so that we can pass them as kwargs to `Message` to get nicer structured logs.